test/ComposerInstallerTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace PhpSchool\WorkshopManagerTest;
+
+use Composer\Factory;
+use PhpSchool\WorkshopManager\ComposerInstaller;
+use PhpSchool\WorkshopManager\Filesystem;
+use PHPUnit_Framework_TestCase;
+use Seld\JsonLint\ParsingException;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Aydin Hassan <aydin@hotmail.co.uk>
+ */
+class ComposerInstallerTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var string
+     */
+    private $tempDir;
+
+    /**
+     * @var Filesystem
+     */
+    private $filesystem;
+
+    public function setUp()
+    {
+        $this->filesystem = new Filesystem;
+        $this->tempDir = sprintf('%s/%s', realpath(sys_get_temp_dir()), $this->getName());
+        @mkdir($this->tempDir, 0777, true);
+    }
+
+    public function tearDown()
+    {
+        $this->filesystem->remove($this->tempDir);
+    }
+
+    public function testComposerOutputIsWrittenIfInVerboseMode()
+    {
+        $input  = new ArrayInput([]);
+        $output = new BufferedOutput();
+        $output->setVerbosity(OutputInterface::VERBOSITY_VERBOSE);
+
+        $installer = new ComposerInstaller($input, $output, new Factory);
+        file_put_contents(sprintf('%s/composer.json', $this->tempDir), '{"name" : "learnyouphp", "require" : { "php": ">=5.6"}}');
+        $res = $installer->install($this->tempDir);
+
+        $this->assertFileExists(sprintf('%s/vendor', $this->tempDir));
+        $this->assertFileExists(sprintf('%s/composer.lock', $this->tempDir));
+
+        $expectedOutput  = "/Loading composer repositories with package information\n";
```

Comment:
Just a thought... are we not making this a fragile test by testing the output of another package? I see why it's done I'm just wondering if it's the best thing to do. 

Unfortunately I don't really have alternative other than checking that there was output which I guess in it self is a pretty weak/fragile test